images/hook/entrypoint.sh

```diff
@@ -70,6 +70,14 @@ if [ $1 = "update" ]; then
         fi
     fi
 
+    # create influxdb secret in case it does not exist because upgading from old gravity version
+    if ! /opt/bin/kubectl --namespace=monitoring get secret influxdb > /dev/null 2>&1;
```

Comment:
nit: do you still need to output to stderr? If not, then `2>&1 >/dev/null `

Reply:
Hmm. What is the difference between mine `> /dev/null 2>&1` and your `2>&1 >/dev/null`? Maybe you just missed that part of the output pipe.

Reply:
No, yours is correct.
The difference is that `2>&1 >/dev/null` will only send stdout to `/dev/null` while `>/dev/null 2>&1` will send both stderr/stdout to `/dev/null`